docs/source/reference/yaml-spec.rst

```diff
@@ -107,6 +107,10 @@ Available fields:
       #
       # default: EAGER_NEXT_REGION
       job_recovery: none
+      # Or, to allow up to 3 retries on failure:
+      # job_recovery:
+      #   strategy: EAGER_NEXT_REGION
+      #   max_retry_on_failure: 3
```

Comment:
Just looking at naming, it's slightly confusing because 'retry' made me think about retry as in autofailover.

How about we call it `max_restarts / max_restarts_on_failure`?

-----

"Or use a dict to additionally specify maximum job restarts:"

Reply:
Good point! Updated. Thanks